### PR TITLE
[OSDOCS-8304] Reinstates AWS and GCP short term creds details

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.14+

Issue:
[OSDOCS-8304](https://issues.redhat.com//browse/OSDOCS-8304)

Link to docs preview:
- [AWS Security Token Service authentication process](https://67750--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-short-term-creds#cco-short-term-creds-auth-flow-aws_cco-short-term-creds)
- [GCP Workload Identity authentication process](https://67750--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-short-term-creds#cco-short-term-creds-auth-flow-gcp_cco-short-term-creds)

QE review:
- [x] QE has approved this change.

Additional information:
- This content was omitted in the reorganization during 4.14, but (unless I am forgetting something) that was unintentional
- Also moves unused topic to archive directory

### CODE REVIEW

**Comment by jstuever** [comment]
lgtm

**Comment by jeana-redhat** [comment]
@jianping-shu I know we talked about this on Slack but wanted to confirm this is verified to add back

**Comment by jianping-shu** [comment]
LGTM

File: modules/cco-short-term-creds-auth-flow-aws.adoc [hunk 1]
```diff
@@ -6,6 +6,8 @@
 [id="cco-short-term-creds-auth-flow-aws_{context}"]
 = AWS Security Token Service authentication process
 
+Requests for new and refreshed credentials are automated by using an appropriately configured AWS IAM OpenID Connect (OIDC) identity provider, combined with AWS IAM roles. {product-title} signs service account tokens that are trusted by AWS IAM, and can be projected into a pod and used for authentication. Tokens are refreshed after one hour.
```
Comment (kalexand-rh) [inline modules/cco-short-term-creds-auth-flow-aws.adoc:9]:
```suggestion
Requests for new and refreshed credentials are automated by using an appropriately configured AWS IAM OpenID Connect (OIDC) identity provider combined with AWS IAM roles. Service account tokens that are trusted by AWS IAM are signed by {product-title} and can be projected into a pod and used for authentication. Tokens are refreshed after one hour.
```

File: modules/cco-short-term-creds-auth-flow-gcp.adoc [hunk 2]
```diff
@@ -6,6 +6,8 @@
 [id="cco-short-term-creds-auth-flow-gcp_{context}"]
 = GCP Workload Identity authentication process
 
+Requests for new and refreshed credentials are automated by using an appropriately configured OpenID Connect (OIDC) identity provider, combined with IAM service accounts. {product-title} signs service account tokens that are trusted by GCP, and can be projected into a pod and used for authentication. Tokens are refreshed after one hour by default.
```
Comment (kalexand-rh) [inline modules/cco-short-term-creds-auth-flow-gcp.adoc:9]: The same suggestions apply here.

You mention "by default" here but not in the AWS version. Is that intentional, or is one way correct for both?

**Review by kalexand-rh — COMMENTED** [review]
I have a clarity issue and a question, but this otherwise LGTM.

File: modules/cco-short-term-creds-auth-flow-gcp.adoc (hunk 2 above)
Comment (jeana-redhat) [reply modules/cco-short-term-creds-auth-flow-gcp.adoc:9]: I do not believe either can be configured by the user here, so probably it should just be "Tokens are refreshed after one hour." @jianping-shu does that make sense to you for GCP?

File: modules/cco-short-term-creds-auth-flow-aws.adoc (hunk 1 above)
Comment (jeana-redhat) [reply modules/cco-short-term-creds-auth-flow-aws.adoc:9]: Thanks! I'm gonna go ahead and open another PR to get this editorial feedback into pre-4.14 (it is not gonna want to cherrypick :sweat_smile:)

File: modules/cco-short-term-creds-auth-flow-gcp.adoc (hunk 2 above)
Comment (jianping-shu) [reply modules/cco-short-term-creds-auth-flow-gcp.adoc]: @jeana-redhat Sorry for missing this question. I think the default duration is 1 hour for token in both AWS and GCP, CCO doesn't configure the duration explicitly and just use the default.
Here is the GCP doc
https://cloud.google.com/docs/authentication/token-types#at-lifetime

Comment (jeana-redhat) [reply modules/cco-short-term-creds-auth-flow-gcp.adoc]: No problem! thanks for the confirmation

**Pull request merged**